### PR TITLE
updated CC structure based on cc modes

### PR DESCRIPTION
**Pull request by Mariangela**
## Purpose
- updated CC structure based on cc modes
- Related to https://github.com/wso2/docs-apim/issues/5014

### CODE REVIEW

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/defining-a-backend-security-scheme.md [hunk 1]
```diff
@@ -20,27 +20,38 @@ When an actual backend service of the API is protected using basic authenticatio
...
   services:
       adapter:
           environment:
           - api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_username="admin"
           - api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_password="admin"
 ```
 
+``` java tab="Kubernetes Example"
+
+TODO
```
Comment (Mariangela) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/defining-a-backend-security-scheme.md:51]: @pubudu538 , please add the correct K8s code snippet

Comment (pubudu538) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/defining-a-backend-security-scheme.md:51]: containers:
        - name: choreo-connect-adapter
          env:
            - name: api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_username
              value: "admin"
            - name: api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_password
              value: "admin"
```suggestion
TODO
```

Comment (pubudu538) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/defining-a-backend-security-scheme.md:51]:
```suggestion
containers:
        - name: choreo-connect-adapter
          env:
            - name: api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_username
              value: "admin"
            - name: api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_password
              value: "admin"
```

Comment (pubudu538) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/defining-a-backend-security-scheme.md:51]:
```suggestion
containers:
    - name: choreo-connect-adapter
      env:
        - name: api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_username
          value: "admin"
        - name: api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_password
          value: "admin"
```

**Review by pubudu538 — APPROVED** [review]

**Pull request merged**